controller-server/src/main/java/com/yahoo/vespa/hosted/controller/persistence/CuratorDb.java

```diff
@@ -123,33 +123,29 @@ public Lock lockOpenStackServerPool() {
     }
 
     public void writeInactiveJobs(Set<String> inactiveJobs) {
-        NestedTransaction transaction = new NestedTransaction();
```

Comment:
I think it's better to keep the transactions in place as it establishes a good pattern for the class. Besides, it's easy to forget when methods are extended to do more than one write.

Reply:
Right, never mind. I thought it was a ZK transaction.